spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/ldap/embedded/EmbeddedLdapAutoConfiguration.java

```diff
@@ -107,6 +109,28 @@ public InMemoryDirectoryServer directoryServer() throws LDAPException {
 					this.embeddedProperties.getCredential().getUsername(),
 					this.embeddedProperties.getCredential().getPassword());
 		}
+
+		Schema schema = null;
+		if (this.embeddedProperties.getValidation().isEnabled()) {
+			schema = Schema.getDefaultStandardSchema();
+			String schemaLocation = this.embeddedProperties.getValidation().getSchema();
+			if (StringUtils.hasText(schemaLocation)) {
```

Comment:
Good point! Should the `spring.ldap.embedded.ldif` property also use a `Resource`? I'm not sure about having two ways to load a resource in this same configuration.